packages/styles/scss/components/date-picker/_date-picker.scss

```
@@ -177,6 +177,7 @@
 
   .#{$prefix}--date-picker__input:disabled ~ .#{$prefix}--date-picker__icon {
     cursor: not-allowed;
+    opacity: 0.3;
```

```suggestion
    fill: $icon-disabled;
```

Can we use the disabled icon token instead?

---

Sure